src/Resources/views/CRUD/action_buttons.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 {% spaceless %}
-    {% for item in admin.getActionButtons(action, object|default(null)) %}
+    {% for item in admin.getActionButtons(action, (object is defined) ? object : null ) %}
```

Comment:
Can you use `??` instead?
https://twig.symfony.com/doc/2.x/templates.html#other-operators

Reply:
In old PR's you can edit it yourself or add a suggestion

Reply:
👍 drop twig 1

Reply:
lets do it in a separate PR, IMO its not worth only because of one `??` 👎